### PR TITLE
Spike support for multiple elements

### PR DESCRIPTION
**Pull request by ahamid**
for what it's worth i went ahead and gave it a shot, tests pass
mostly for review/considering, not literal pull request ;)

* add support for option.elements
* option.element coerced to this.elements array
* this.element preserved for backward compatibility/simplicity
* update context matching (any element in this.elements matches)
* ListUpdater not updated to support multiple elements

### CODE REVIEW

File: Updater.js [hunk 1]
```diff
@@ -59,7 +67,7 @@ define(['./util/lang', './Variable'], function (lang, Variable) {
 		},
 		updated: function (updateEvent, by, context) {
 			if (!this.invalidated) {
-				if (!context || context == this.element || context.contains(this.element)) {
+				if (!context || this.contextMatches(context)) {
```
Comment (ahamid) [inline Updater.js]: could probably be improved by returning the matched subset to limit extraneous renders

File: Updater.js [hunk 2]
```diff
@@ -259,7 +278,7 @@ define(['./util/lang', './Variable'], function (lang, Variable) {
 	ListUpdater.prototype.renderUpdate = function (newValue, element) {
 		var container
 		var each = this.each
-		var thisElement = this.element
+		var thisElement = this.elements[0]
```
Comment (ahamid) [inline Updater.js]: oops, `this.element` should suffice. in any case, probably not worthwhile to support multiple targets for lists...

**Comment by kriszyp** [comment]
What does "spike" support mean? This gives the ability to defining multiple elements on an Updater?

**Comment by kriszyp** [comment]
Oh, I see, this is in regards to issue #9, that makes sense now.

**Comment by kriszyp** [comment]
I am not sure why some merges aren't detected, but merged in https://github.com/kriszyp/alkali/commit/630ec79ae949f2006033216cf6a0724116af21a2

**Pull request closed without merging**